app/helpers/operatingsystems_helper.rb

```diff
@@ -27,8 +27,8 @@ def icon(record, opts = {})
              "scientific"
            when /archlinux/i
              "archlinux"
-           when /altlinux/i
-             "stub/goldenrod-a"
+           when /alt/i
+             "alt"
```

Comment:
Can you move the when statement to the very bottom? "alt" is a quite short word part, it will probably match with many words that people can use for OS names. So let's have it in the end as a precaution.

Reply:
I'm sure not, could you provide any example?